opensrp-reveal/src/main/java/org/smartregister/reveal/fragment/AvailableOfflineMapsFragment.java

```diff
@@ -0,0 +1,247 @@
+package org.smartregister.reveal.fragment;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.content.LocalBroadcastManager;
+import android.support.v7.widget.RecyclerView;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.mapbox.mapboxsdk.geometry.LatLng;
+
+import org.smartregister.reveal.BuildConfig;
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.adapter.OfflineMapAdapter;
+import org.smartregister.reveal.contract.AvailableOfflineMapsContract;
+import org.smartregister.reveal.model.OfflineMapModel;
+
+import java.util.List;
+
+import io.ona.kujaku.helpers.OfflineServiceHelper;
+import io.ona.kujaku.services.MapboxOfflineDownloaderService;
+import io.ona.kujaku.utils.Constants;
+
+public class AvailableOfflineMapsFragment extends Fragment implements AvailableOfflineMapsContract.View {
+
+    private RecyclerView offlineMapRecyclerView;
+    private OfflineMapAdapter adapter;
+    private Button downloadMapButton;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'downloadMapButton' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)